backend/src/main/java/io/debezium/configserver/rest/ConnectorURIs.java

```diff
@@ -0,0 +1,18 @@
+package io.debezium.configserver.rest;
```

Comment:
All files should have the license header as in Debezium core. Probably needs adding the right CS check, too.

Reply:
No file has this yet. I created https://issues.redhat.com/browse/DBZ-2985 to do this (add headers and CS check) for the whole project.

Reply:
Ok, thanks, Would be good to do this one quickly.